### DISCUSSION TITLE
Newline is not properly displayed in a comment

### DISCUSSION THREAD

**Discussion by LevinGermann** [directus/directus#13322] [Archive]
### Describe the Bug

Seems like the newline character is not properly rendered in the comments. I'm pretty sure that this used to work on 9.4.1 (not sure about later versions).

https://user-images.githubusercontent.com/9347206/168581772-13c66774-6496-4cd2-871c-de6158d46338.mov

In the directus_activity table the comment is correctly saved with the newlines.

![image](https://user-images.githubusercontent.com/9347206/168582118-08180487-b121-441b-ae27-d44498f64522.png)

### To Reproduce

Save a comment with newlines

### What version of Directus are you using?

9.10.0

### What version of Node.js are you using?

v16.7.0

### What database are you using?

PostgresQL 13.2

### What browser are you using?

Chrome

### How are you deploying Directus?

Docker

**Comment by br41nslug** [comment]
Back in 9.4.1 Directus did indeed render comment content differently. Since then comments are now Markdown based and Markdown does ignore the first line break usually, a newline can be forced by using 2 breaks or adding a `\` at the end of the line.

**Reply by LevinGermann** [reply]
Thanks for the explanation!
